docker/package/package_generator.py

```diff
@@ -167,7 +169,7 @@ def gen_makefile(pkg: Package, out):
         f.write(makefile_contents)
 
 def gen_changelog(pkg: Package, ubuntu_version, maintainer, date, out):
-    changelog_contents = f'''{pkg.get_full_name().lower()} ({package_version}) {ubuntu_version}; urgency=medium
+    changelog_contents = f'''{pkg.get_full_name().lower()} ({ubuntu_epoch}:{version}-0ubuntu{release}) {ubuntu_version}; urgency=medium
```

Comment:
Btw, does `0` in `0ubuntu` have any meaning?

Reply:
https://www.ducea.com/2006/06/17/ubuntu-package-version-naming-explanation/

Reply:
> 2006

🧓 

But the article is good, thanks.